salt/utils/pkg/rpm.py

```diff
@@ -59,16 +60,17 @@ def check_32(arch, osarch=None):
     return all(x in ARCHES_32 for x in (osarch, arch))
 
 
-def pkginfo(name, version, arch, repoid):
+def pkginfo(name, version, arch, repoid, install_date, install_date_time_t):
```

Comment:
Seems pkginfo is only used by
salt/salt/modules/yumpkg.py:                pkginfo = salt.utils.pkg.rpm.pkginfo(**cur)
Have you followed the code through to check the impact of add extra parms and returning extra data?